lecsum-fe/services/api_client.py

```
@@ -97,3 +97,27 @@ def grade_quiz(self, quiz_set_id: int, quiz_id_list: list[int], user_answer_list
+        params = {
+            "quiz_set_id": quiz_set_id,
+            "limit": limit,
+            "offset": offset
+        }
+        response = requests.get(f"{self.base_url}/api/quizzes/attempts", params=params)
+        response.raise_for_status()
+        return response.json()
+
+    def get_attempt_detail(self, attempt_id: int) -> Dict[str, Any]:
+        """응시 기록 상세 조회 (문제별 결과 포함)"""
+        response = requests.get(f"{self.base_url}/api/quizzes/attempts/{attempt_id}")
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)